src/ucp/wireup/select.c

```diff
@@ -16,6 +16,17 @@
 
 #define UCP_WIREUP_RMA_BW_TEST_MSG_SIZE       262144
 
+#define UCP_WIREUP_CHECK_AMO_FLAGS(_ae, _criteria, _context, _op, _size)                  \
+    if (!ucs_test_all_flags(_ae->iface_attr.atomic.atomic##_size._op##_flags,             \
+                            _criteria->remote_atomic_flags.atomic##_size._op##_flags)) {  \
+        ucs_trace("addr[%d] %s: no %s", addr_index,                                       \
+                  ucp_find_tl_name_by_csum(_context, _ae->tl_name_csum),                  \
```

Comment:
need to wrap arg usage with ( )
addr_index should also be an argument

Reply:
wrapped, addr_index is added